Global database and summary sheet.R

```diff
@@ -5,43 +5,53 @@
 #
 ######################################################################################################
 
-# LOAD PACKAGES
+# LOAD PACKAGES ----
```

Comment:
Dashes were added to section heading comments to turn them into headers, and to thereby enable a navigation outline